src/main/java/com/okta/tools/aws/settings/Credentials.java

```diff
@@ -60,21 +59,8 @@ public Credentials(Reader reader) throws IOException {
      */
     public void addOrUpdateProfile(String name, String awsAccessKey, String awsSecretKey, String awsSessionToken) {
         name = "default".equals(name) ? "default" : name + environment.credentialsSuffix;
```

Comment:
Should this also be using `DEFAULT_PROFILE_NAME` instead?

Reply:
Definitely. I’ll fix that. Good catch.